unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/DCLVideoTexture.cs

```diff
@@ -140,9 +146,15 @@ public override IEnumerator ApplyChanges(BaseModel newModel)
                 }
 
                 if (model.playing)
+                {
                     texturePlayer.Play();
+                    ReportVideoProgress();
+                }
                 else
+                {
                     texturePlayer.Pause();
+                    ReportVideoProgress();
+                }
```

Comment:
I would pay attention to this method, probably calling `ReportVideoProgress` at the end at every change may be a good idea.

```suggestion
                if (model.playing)
                {
                    texturePlayer.Play();
                }
                else
                {
                    texturePlayer.Pause();
                }
                ReportVideoProgress();
```

Reply:
Yes you are right, it made sense last time when they were different